app/components/youtube/client.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const youtubeVideoPlayer = require('../../services/universal/youtube-video-player'),
+  { Visible, isElementNotHidden } = require('../../services/client/visibility');
+
+module.exports = (el) => {
+  let autoplay = el.getAttribute('data-autoplay-video') === 'true',
```

Comment:
Make these properties `const`
```suggestion
  const autoplay = el.getAttribute('data-autoplay-video') === 'true',
```